ans-abstract/abstract.tex

```diff
@@ -49,21 +49,21 @@
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 \section{Introduction}
 
-Climate change presents a threat that we should address swiftly. Decarbonizing the electric grid through variable renewable energy and nuclear power seems to be a remedy. Unfortunately, a carbon neutral electric grid will be insufficient to halt climate change because transportation contributes more to \gls{GHG} emissions than electricity. As seen in Figure \ref{fig:ghg}, transportation produced the most \glspl{GHG} in the US in 2017. Thus, decarbonizing transportation underpins global carbon reduction. Accordingly, the \gls{UIUC} has committed to the Illinois Climate Action Plan (iCAP) which aims to attain carbon neutrality by 2050 \cite{noauthor_illlinois_2015}.
+Climate change presents a threat that we should address swiftly. Decarbonizing the electric grid through variable renewable energy and nuclear power seems to be a remedy. Unfortunately, a carbon neutral electric grid will be insufficient to halt climate change because transportation contributes more to \gls{GHG} emissions than electricity. As seen in Figure \ref{fig:ghg}, transportation produced the most \glspl{GHG} in the US in 2017. Thus, decarbonizing transportation underpins global carbon reduction. Accordingly, the \gls{UIUC} has committed to the Illinois Climate Action Plan (iCAP) which aims to attain carbon neutrality by 2050 \cite{university_of_illinois_at_urbana-champaign_illlinois_2015}.
```

Comment:
Illinois is spelled wrong in the citation ``\cite{...illlinois}``
I'm not sure how you generated the bibliography, you may want to check the Zotero metadata.

Reply:
good catch